VContainer/Assets/VContainer/Runtime/Container.cs

```diff
@@ -110,6 +114,8 @@ public bool TryGetRegistration(Type type, out Registration registration)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public void Dispose()
         {
+            EmitDisposeCallbacks();
```

Comment:
Is there any way to add values to disposables instead of holding dispoeCallbacks?

For example.

```cs
struct ActionDisposable : IDisposable
{
    Action calback;

    public ActionDisposable(Action callback)
    {
        this.callback = callback; }
    }

   public void Dispose() => callback();
}

disposables.Add(new ActionDisposable(callback));
````


Translated with www.DeepL.com/Translator (free version)